test/quantization/dbr/test_quantize_dbr.py

```diff
@@ -118,40 +118,49 @@ class TestQuantizeDBRIndividualOps(QuantizeDBRTestCase):
     Tests that DBR quantization covers individual ops
     """
     def test_conv(self):
+        convs = {1: nn.Conv1d, 2: nn.Conv2d, 3: nn.Conv3d}
+
         class M(torch.nn.Module):
-            def __init__(self):
+            def __init__(self, dim):
                 super().__init__()
-                self.conv = torch.nn.Conv2d(1, 1, 1)
+                self.conv = convs[dim](3, 3, 3)
 
             def forward(self, x):
-                x1 = self.conv(x)
-                return x1
+                return self.conv(x)
 
-        m = M().eval()
-        qconfig = torch.quantization.default_qconfig
-        self._test_auto_tracing(m, qconfig, (torch.randn(1, 1, 2, 2),))
+        for dim in range(1, 4):
+            m = M(dim).eval()
+            qconfig = torch.quantization.default_qconfig
+            data = self.img_data_dict[dim][0][0]
```

Comment:
nit: could we use something like `torch.randn` instead of `self.img_data_dict`?  this is a personal preference, it's easier to use these tests for debugging if the all the information is specified inline and we don't have to navigate to utils files and base classes to understand the structure of the input data

Reply:
just curious, thoughts about my comment above?

Reply:
Oh sorry I missed this one